velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -14,12 +14,18 @@
  * limitations under the License.
  */
 #include <gtest/gtest.h>
+#include "velox/exec/tests/utils/HiveConnectorTestBase.h"
 #include "velox/exec/tests/utils/PlanBuilder.h"
 #include "velox/functions/prestosql/registration/RegistrationFunctions.h"
 #include "velox/parse/TypeResolver.h"
+#include "velox/type/tests/FilterBuilder.h"
+#include "velox/type/tests/SubfieldFiltersBuilder.h"
 #include "velox/vector/tests/VectorTestBase.h"
 
 using namespace facebook::velox;
+using namespace facebook::velox::common::test;
+using namespace facebook::velox::exec::test;
```

Comment:
Got an error:

There are multiple using directives that potentially have name conflicts or are redundant.
This using test directive has potential name conflicts with the using test directive at velox/exec/tests/PlanNodeToStringTest.cpp:26:42.
Redundant using declarations may be a sign of code that should be checked for errors.